Resources/Prototypes/Entities/Structures/Machines/artifact_analyzer.yml

```diff
@@ -42,10 +42,8 @@
   - type: ApcPowerReceiver
     powerLoad: 12000
     needsPower: false #only turns on when scanning
-  - type: UpgradePowerDraw
-    powerDrawMultiplier: 0.80
-    scaling: Exponential
   - type: ArtifactAnalyzer
+    analysisDuration: 30
```

Comment:
just change the default value of the datafield in c# if this is supposed to be the default value

Reply:
shit good point